app/Jobs/Curations/AddStatus.php

```diff
@@ -61,7 +61,9 @@ public function handle()
 
     private function isCurrentStatus()
     {
-        return $this->hasCurationCurationStatus() && $this->curation->curationStatuses->first()->id == $this->curationStatus->id;
+        return $this->hasCurationCurationStatus() && 
+                $this->curation->curationStatuses->first()->id == $this->curationStatus->id && 
```

Comment:
Does laravel cache the calls to `$this->curation->curationStatuses->first()`? If not, we could use a local variable to prevent extra db calls.